starport/services/network/init.go

```diff
@@ -88,6 +88,16 @@ func (b *Blockchain) InitAccount(ctx context.Context, v chain.Validator, account
 
 // initGenesis creates the initial genesis of the genesis depending on the initial genesis type (default, url, ...)
 func (b *Blockchain) initGenesis(ctx context.Context) error {
+	genesisPath, err := b.chain.GenesisPath()
+	if err != nil {
+		return err
+	}
+
+	// remove existing genesis
+	if err := os.Remove(genesisPath); err != nil {
```

Comment:
if the file is already deleted manually by the user, this method will spawn an error

Reply:
```suggestion
	if err := os.RemoveAll(genesisPath); err != nil {
```

Reply:
```suggestion
	if err := os.RemoveAll(genesisPath); err != nil {
```